Jarvis-Platform/jarvis-studio
# Feature/Table to storage run view

# Whats new?
- `table-to-storage` run view
- `get-item-mixin` to request info of an item depending id parameter in URL
- Data of the run view are now defined in `view` while it was in `view-item` before (=> Has to refacto other components like this in the future)

# How to test
- Go here => `/table-to-storage/runs`
- Click on an item
- Try to load different tabs

⚠ As I changed some components behavior with props, try to load different pages of data-workflows with the browser console open to see if there is errors on specific cases.

@romone22 I didn't know which data I have to put in the `Configuration` tab, I probably have to update this one, keep me update :)
